### PR TITLE
[ZT] Access custom pages

### PR DESCRIPTION
**Pull request by ranbel**

### CODE REVIEW

File: content/cloudflare-one/applications/app-launcher.md [hunk 1]
```diff
@@ -33,3 +31,25 @@ To enable the App Launcher:
 5. Select **Save**.
 
 The App Launcher is now available at `<your-team-name>.cloudflareaccess.com`. You can always edit your App Launcher rules by going to **Settings** > **Authentication**.
+
+## Add a tile to the App Launcher
+
+Tiles have a one-to-one relationship with each application you create in Access. For example, if you create one application for general access to your Jira deployment and a separate application that restricts requests to a particular Jira path, a user authorized for both will see separate tiles for each. The tile names displayed in the Access App Launcher portal correspond to the application names listed under **Access** > **Applications**.
```
Comment (kennyj42) [inline content/cloudflare-one/applications/app-launcher.md]: should we call out multi-hostname here?

**Review by kennyj42 — APPROVED** [review]

File: content/cloudflare-one/applications/app-launcher.md (hunk 1 above)
Comment (ranbel) [reply content/cloudflare-one/applications/app-launcher.md]:
```suggestion
Tiles have a one-to-one relationship with each application you create in Access. For example, if you create one application for general access to your Jira deployment and a separate application that restricts requests to a particular Jira path, a user authorized for both will see separate tiles for each. If you add multiple hostnames to a single application, the user will only see a single tile. The tile names displayed in the Access App Launcher portal correspond to the application names listed under **Access** > **Applications**.
```

Comment (ranbel) [reply content/cloudflare-one/applications/app-launcher.md]: Good idea. How about something like this?

Comment (kennyj42) [reply content/cloudflare-one/applications/app-launcher.md]: If you add multiple hostnames (hyperlink to multi-domain apps section) to a single application, the user will only see the domain selected in Application Appearance.

**Pull request merged**